### DISCUSSION TITLE
Send non ethereum

### DISCUSSION THREAD

**Discussion by Vadim51515** [WalletConnect/walletconnect-monorepo#1751] [Developer Support]
Hi, using the trust app and @walletconnect, I was able to transfer ether. Can I transfer another currency in the same way? (e.g. usdt )

**Comment by nikitaNotFound** [comment]
Hi @Vadim51515! Any results with your question?

**Reply by Vadim51515** [reply]
@nikitaNotFound Привет, к сожалению нет, у меня так и не получилось найти информацию по этому вопросу
